bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -237,6 +237,14 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
         case _                 => false
       }
 
+    def bpIsGnirsLRIFU(b: BlueprintBase): Boolean =
+      b match {
+        case GnirsBlueprintSpectroscopy(AltairNone, _,  _, _, GnirsFpu.LR_IFU, _)           => false
+        case GnirsBlueprintSpectroscopy(AltairLGS(true, _, _), _, _, _, GnirsFpu.LR_IFU, _) => false
```

Comment:
On the surface of it, since it is `GnirsFpu.LR_IFU`, I would expect this to be `true` for these two cases as well.  Maybe a comment about what is going on here?

Reply:
I'll add a comment but basically LR_IFU is only compatible with altair None or Altair LGS/PWFS1